frontend/src/components/Form/Form.js

```diff
@@ -6,73 +6,140 @@ import routePaths from '../../constants/routePaths';
 const Form = (props) => {
   const { formType, handleSubmit, inputsArr } = props;
 
+  const categoriesArr = props.categoriesArr;
```

Comment:
please don't abbreviate (or at least try to avoid the habit as much as possible)

Reply:
fixed